samples/httpd/main.c

```diff
@@ -4,7 +4,7 @@
 #include "lwip/netif.h"
 #include "lwip/sys.h"
 #include "lwip/tcpip.h"
-#include "lwip/timers.h"
+#include "lwip/timeouts.h"
```

Comment:
http://git.savannah.nongnu.org/cgit/lwip.git/commit/?id=5d4c1432c2b223f2e5a2511416a2cc7fdff3a61c renamed this header to remove a naming clash on FreeRTOS.